### PR TITLE
docs: reorganized to have all IDE options in single doc. Added VS for…

### PR DESCRIPTION
**Pull request by mkwhalen**
… Mac

GitHub Issue (If applicable): #

## PR Type

What kind of change does this PR introduce?

- Documentation content changes

### CODE REVIEW

File: doc/articles/get-started.md [hunk 1]
```diff
@@ -57,6 +64,114 @@ If XAML Intellisense isn't working on a freshly-created project, try the followi
...
+1. In the terminal type the following to create a new project:
+```bash
+dotnet new unoapp -o MyApp -ios=false -android=false -macos=false -uwp=false --vscodeWasm
+```
+
+This will create a solution that only contains the WebAssembly platform support.
+
+### Prepare the WebAssembly application for debugging
+
+1. Install the [C# extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp) and the [JavaScript Debugger (Nightly)](https://marketplace.visualstudio.com/items?itemName=ms-vscode.js-debug-nightly) extension with the debug.javascript.usePreview setting set to true (**File** / **Preference** / **Settings**, search for `Use preview`).
```
Comment (davidjohnoliver) [inline doc/articles/get-started.md:89]:
```suggestion
1. Install the [C# extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp) and the [JavaScript Debugger (Nightly)](https://marketplace.visualstudio.com/items?itemName=ms-vscode.js-debug-nightly) extension with the `debug.javascript.usePreview` setting set to true (**File** / **Preference** / **Settings**, search for `Use preview`).
```

File: doc/articles/get-started.md [hunk 2]
```diff
@@ -57,6 +64,114 @@ If XAML Intellisense isn't working on a freshly-created project, try the followi
...
+1. Open Code using
+    ```bash
+    code ./MyApp`
+    ```
+1. Visual Studio will ask to restore the nuget packages
```
Comment (davidjohnoliver) [inline doc/articles/get-started.md:94]:
```suggestion
1. Visual Studio Code will ask to restore the NuGet packages.
```

File: doc/articles/get-started.md [hunk 3]
```diff
@@ -57,6 +64,114 @@ If XAML Intellisense isn't working on a freshly-created project, try the followi
...
+
+1. Starting the app with the WebAssembly debugger is a two-step process:
+    1. Start the app first using the **�.NET Core Launch (Uno Platform App)�** launch configuration
+    1. Then start the browser using the **�.NET Core Debug Uno Platform WebAssembly in Chrome�** launch configuration (requires Chrome). To use the latest stable release of Edge instead of Chrome, change the type of the launch configuration in `.vscode/launch.json` from `pwa-chrome` to `pwa-msedge`
+1. Place a breakpoint in the OnClick method
+1. Click the button in the app, and the breakpoint will hit
+
+### Updating an existing application to work with VS Code
+
+If you already have an Uno application, you can add some some missing support files the VS Code to recognize your project.
```
Comment (davidjohnoliver) [inline doc/articles/get-started.md:127]:
```suggestion
If you already have an Uno application, you can add some some missing support files for VS Code to recognize your project.
```

File: doc/articles/get-started.md [hunk 4]
```diff
@@ -57,6 +64,114 @@ If XAML Intellisense isn't working on a freshly-created project, try the followi
...
+* An [**Apple ID**](https://support.apple.com/en-us/HT204316)
+* .NET Core 2.2
+
+### Modifying Existing Uno App
+
+1. Open project in Visual Studio for Mac
+ ![new-project](Assets/quick-start/vs-mac-open-project.png)
+Once open, you should see your folder structure set up like this:
+![folder-structure](Assets/quick-start/vs-mac-folder-structure.png)\
+If you have a warning symbol on your iOS project, make sure you have the minimum version of Xcode.
```
Comment (davidjohnoliver) [inline doc/articles/get-started.md:155]:
```suggestion
If you have a warning symbol on your iOS project, make sure you have the minimum version of XCode.
```

File: doc/articles/get-started.md [hunk 5]
```diff
@@ -57,6 +64,114 @@ If XAML Intellisense isn't working on a freshly-created project, try the followi
...
+
+1. Install the [C# extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp) and the [JavaScript Debugger (Nightly)](https://marketplace.visualstudio.com/items?itemName=ms-vscode.js-debug-nightly) extension with the debug.javascript.usePreview setting set to true (**File** / **Preference** / **Settings**, search for `Use preview`).
+1. Open Code using
+    ```bash
+    code ./MyApp`
+    ```
+1. Visual Studio will ask to restore the nuget packages
+
+### Modify the template
+1. In MainPage.xaml, replace the Grid's content with the following:
```
Comment (davidjohnoliver) [inline doc/articles/get-started.md:97]:
```suggestion
1. In `MainPage.xaml`, replace the Grid's content with the following:
```

File: doc/articles/get-started.md [hunk 6]
```diff
@@ -57,6 +64,114 @@ If XAML Intellisense isn't working on a freshly-created project, try the followi
...
+    <StackPanel> 
+        <TextBlock x:Name="txt" 
+                    Text="Hello, world!" 
+                    Margin="20" 
+                    FontSize="30" /> 
+        <Button Content="click" 
+                Click="OnClick" /> 
+    </StackPanel>
+    ```
+1. In your MainPage.xaml.cs, add the following method:
```
Comment (davidjohnoliver) [inline doc/articles/get-started.md:108]:
```suggestion
1. In your `MainPage.xaml.cs`, add the following method:
```

**Comment by mkwhalen** [comment]
Any incite on why some of the CI pipelines are failing?

**Pull request merged**